### PR TITLE
feat: adding Azure Container Instances module

### PR DESCRIPTION
**Pull request by polatengin**
This PR is for adding Azure Container Instances module to the Terratest.

With this module, it can be possible to test Azure Container Instance resources on Terraform scripts.

### CODE REVIEW

File: examples/azure/terraform-azure-aci-example/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+> This was tested with **go1.14.1**.
+
+### Check Azure-sdk-for-go version
+
+Let's make sure [go.mod](https://github.com/gruntwork-io/terratest/blob/master/go.mod) includes the appropriate [azure-sdk-for-go version](https://github.com/Azure/azure-sdk-for-go/releases/tag/v38.1.0):
+
+```go
+require (
+    ...
+    github.com/Azure/azure-sdk-for-go v38.1.0+incompatible
```
Comment (rguthriemsft) [inline examples/azure/terraform-azure-aci-example/README.md:49]: bump to 46

**Comment by HadwaAbdelhalem** [comment]
Hi @yorinasub17 , Please take a look when you get a chance.  [Here](https://github.com/polatengin/terratest/runs/3264563922?check_suite_focus=true) is  a link to the related passed CI.

**Review by yorinasub17 — APPROVED** [review]
LGTM!

**Comment by yorinasub17** [comment]
I'll kick off a sanity check build and if it passes our precommit hooks, will merge.

**Comment by yorinasub17** [comment]
The build passed the precommit phase, so will merge and release now!

**Pull request merged**